api/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""Backend api server that is used in the MOH project.
+
+A simple flask backend that will maintaine a live queue of current students in office hours and keep
+track of current student status in the within the queue
+by making request updating the `vist` table within the external postgrest database.
+"""
+
+# TODO: Make the doc string sound better
+
+
+import datetime
+from flask import Flask, request
+
+
+def create_app():
+    """Create and return flask api server
+
+    This funciton is used to setup the flask api server loading all it's depended modulars
```

Comment:
Use proper grammar/spelling, including capitalization. I won't nitpick all instances, but this sentence has 7 errors.

```suggestion
    This function is used to set up the Flask API server loading all its dependent modules
```